terraform/udp/healthcare-oie/healthcare-oie_tf_12_9.tf

```diff
@@ -0,0 +1,151 @@
+variable "org_name" {}
+variable "api_token" {}
+variable "base_url" {}
+variable "demo_app_name" { default = "healthcare" }
+variable "udp_subdomain" { default = "local_zartan" }
```

Comment:
Shouldn't this default simply be "local"?

Reply:
Good find!